app/dashboard/migrations/0174_profile_google_user_id.py

```diff
@@ -7,7 +7,7 @@
 def populate_google_id(apps, schema_editor):
     from dashboard.models import Profile
 
-    profiles_with_google_connection = Profile.objects.filter(is_google_verified=True)
+    profiles_with_google_connection = Profile.objects.filter(is_google_verified=True).only('identity_data_google', 'google_user_id')
```

Comment:
I have to change this, accessing `is_poh_verified` and `poh_handle` before the migration (app/dashboard/migrations/0176_poh_fields.py) causes this to fail.

Reply:
updating an older migration might have side effects on the live server cause this would have already been executed. Is there a way we could do without this ?

Reply:
I think this change is safe because does not change any field on DB side.
Also without this change, adding any further filled to `Profile` causes migrations to fail in this line.